qtodotxt/ui/controllers/main_controller.py

```diff
@@ -383,7 +383,15 @@ def revert(self):
     def openFileByName(self, filename):
         logger.debug('MainController.openFileByName called with filename="{}"'.format(filename))
         self._fileObserver.clear()
-        self._file.load(filename)
+        try:
+            self._file.load(filename)
+        except:
+            currentfile = self._settings.value("last_open_file", "")
+            if currentfile == filename:
+                self._dialogs.showError(self.tr("Current file '{}' was deleted".format(filename)))
+            else:
+                self._dialogs.showError(self.tr("Error opening file: {}".format(filename)))
```

Comment:
in linux i get message aboute file could'n open in win - not, i test in linux and correct...